### ISSUE TITLE
npm run build building with system url path

### ISSUE THREAD

**Issue by jordaofranca** [developit/preact-cli#7]
When I run `npm run build` the sw.js is generated with system paths to the archives:

"C:/Users/jordao/projects/compra/build/1.chunk.82189a4e17c69863082e.js"

instead of:

"1.chunk.82189a4e17c69863082e.js"

**Comment by developit** [comment]
Ah yeah. I think something needs to tell the sw plugin what webpacks context directory was.

**Comment by stevenfitzpatrick** [comment]
I remember using "stripPrefixMulti" in sw-precache plugin to fix this

new SWPrecacheWebpackPlugin({
    cacheId: pkg.name,
    filename: 'sw.js',
    minify: true,
    maximumFileSizeToCacheInBytes: 4194304,
    stripPrefixMulti: {
      'C:/dev/site/dist': '',
      '/home/travis/build/myname/site/dist': ''
    }
  }),

**Comment by developit** [comment]
I think we can even just pass it `process.cwd()`

**Comment by lukeed** [comment]
You can also access webpack's `root`, although I can't remember what it's called now.

**Comment by developit** [comment]
yup yup. just figured we'd keep this bit generic since there might be a time when we want to support an "eject" feature.

**Comment by jordaofranca** [comment]
Can we change it to this?

```
new SWPrecacheWebpackPlugin({
		filename: 'sw.js',
		navigateFallback: 'index.html',
		minify: true,
		staticFileGlobsIgnorePatterns: [
			/\.map$/,
			/push-manifest\.json$/
		],
		stripPrefixMulti: {
      [process.cwd()]: ''
    }
})
```

**Comment by NekR** [comment]
Oh, if there only was a plugin which can do all that automatically. It would be such a bless 😎 😂

**Comment by developit** [comment]
I believe this was fixed in `1.1.1`?

**Comment by rkostrzewski** [comment]
Indeed it was, absolute paths are no longer there. Closing.
